app/frontend/components/common/navbar/navbarUnauth.js

```diff
@@ -45,14 +47,14 @@ const NavbarUnauth = connect(
         h(
           'button',
           {
-            class: 'demo-button rounded-button',
+            class: 'button outline navbar-button',
```

Comment:
Please, remove `navbar-button`. I thought you wrote that we will have 2 types of buttons. Normal and small. I will stick to it at the moment. If you want to have another version of the `.button`, it should be defined next to the `.button` declarations, not next to the `.nav`.

Reply:
We discussed that we won't have 3 versions for the "common" button as intended by the design but we'll simplify that for now by having just 2 common button versions. The "special" button that we've discussed is the navbar button which remains intact in terms of styling so  I'll move the `navbar-button` styling to the buttons portion of the file.